api_tests/actions/views/test_action_list.py

```diff
@@ -41,6 +43,24 @@ def provider(self):
     def node_admin(self):
         return AuthUserFactory()
 
+    @pytest.fixture()
+    def preprint_with_draft_node(self, moderator, provider):
+        draft_node = DraftNodeFactory(creator=moderator)
```

Comment:
Isn't the case that causes problems when a Preprint doesn't have a supplemental node? I didn't think DraftNodes were part of the preprint workflow at all, just the Registrations workflow.

Reply:
Ok I've changed the test to reflect this.